HousingManagementSystemApi/Constants.cs

```diff
@@ -1,5 +1,9 @@
 namespace HousingManagementSystemApi;
 
+using System.Collections.Generic;
```

Comment:
Without any other code changes in this file, these new using statements seem redundant.
Please review and remove if they aren't necessary.